README.md

```diff
@@ -1,2 +1,17 @@
-# GRASS_Web
-Hugo website for GRASS GIS
+# Hugo-based website for GRASS GIS
+
+## How to install
+
+### Debian
+
+    apt install hugo
+
+## How to build
+
+Run in the `grass-website` root directory:
+
+    hugo
+
+Open web site (using Mozilla Firefox in this case):
+
+    firefox public/index.html
```

Comment:
If previous suggestion will be accepted this line should be changed as

```suggestion
    firefox http://localhost/grass-website
```

Reply:
I update how to run hugo server locally. @lucadelu Feel free to open new PR for running website on production server.